test/simapp/test_helpers.go

```diff
@@ -72,22 +74,21 @@ func SetupWithGenesisValSet(valSet *tmtypes.ValidatorSet) *SimApp {
 	stateBytes, _ := json.MarshalIndent(genesisState, "", " ")
 
 	// init chain will set the validator set and initialize the genesis accounts
-	app.InitChain(
-		abci.RequestInitChain{
+	_, err := app.InitChain(
+		&abci.RequestInitChain{
 			Validators:      []abci.ValidatorUpdate{},
 			ConsensusParams: DefaultConsensusParams,
 			AppStateBytes:   stateBytes,
 		},
 	)
+	if err != nil {
+		panic(err)
+	}
 
-	// commit genesis changes
```

Comment:
(Baseapp) BeginBlock and EndBlock are now internal to baseapp. For testing, user must call `FinalizeBlock`